exams/api_test.py

```diff
@@ -257,6 +257,78 @@ def test_update_authorizations_for_exam_run(self):
             assert auth.status == ExamAuthorization.STATUS_PENDING
             assert auth.operation == ExamAuthorization.OPERATION_UPDATE
 
+    def test_update_authorizations_for_past_exam_run(self):
```

Comment:
Instead of adding these two tests can you parameterize the previous test to handle the three cases?